src/DynamoMLDataPipeline/DynamoMLDataPipeline.cs

```diff
@@ -184,6 +186,8 @@ static void DataExchangeToForge(string filePath, RestClient client, string token
             // from the response - these will be consumed by the following API calls.
             ExchangeContainerId = exchangeRequestResponseBody["id"].Value;
 
+            Analytics.TrackEvent(Actions.SendWorkspaceToDynamoML, Categories.DynamoMLDataPipelineOperations, "ExchangeContainerID", Convert.ToInt32(ExchangeContainerId));
```

Comment:
maybe just use the action `export`? As long as the combination of category and action makes sense and is meaningful, we don't need to specify the action too much IMHO